toolbox/cijoe-pkg-xnvme/src/cijoe/xnvme/tests/logic/test_xnvme_tests_xnvme_file.py

```diff
@@ -0,0 +1,24 @@
+"""
+New addition
```

Comment:
This does not seem to be very relevant.
I think it would be better to add a description of the test

Reply:
removed / fixed.